src/main/java/com/google/events/cloud/cloudbuild/v1/StepTimeout.java

```diff
@@ -23,7 +23,7 @@
  */
 public class StepTimeout {
```

Comment:
If we get a new class every time a google.protobuf.Duration is used, that's going to be a terrible user experience.

Reply:
Started an email thread to talk about this concern.